src/sas/qtgui/Perspectives/Invariant/UI/InvariantDetailsUI.ui

```diff
@@ -89,6 +89,9 @@
         <property name="toolTip">
          <string>Extrapolated invariant from low-Q range.</string>
         </property>
+        <property name="readOnly">
+         <bool>true</bool>
+        </property>

```

Comment:
On Mac there is no visual indication that a field is read-only.  You click and either a cursor does or does not appear. When it doesn't you click a bunch of times to make sure you didn't miss the entry field when clicking.

Maybe set the background for read-only fields to the same color as the form so they look like the labels that they are?

Reply:
Sadly that is true for PC also.  This is the quick improvement over all those fields being editable making the user think they were changing something when in fact the code was ignoring them.

I can also disable the box which will make it gray like the background but then the text can no longer be copied to the clipboard and more annoyingly it is grey as well instead of black which I personally find more annoying that the current.  As I stated somewhere (maybe in the commit message? if so that would be pointless since nobody reads those 😃  I do not know yet how to achieve what we had in 4.x.  Moreover I think this is a general thing in the ESS_GUI and standardizing output boxes would be I think the right answer - but that requires a discussion I think.